open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -1,19 +1,131 @@
+makes sense. See [this section](#responding-to-out-of-scope-submissions) about 
+how to respond to out-of-scope (pre-) submissions.
+- Responds to review referrals from JOSS or other publication partners.
+- Monitors pace of review process and reminds other editors to move packages along as needed.
+- Requests a new EiC when their rotation is up (set a calendar reminder ahead of your expected end date and ask for volunteers in the editors’ Slack channel)
+
+This Editor in Chief position rotates between different pyOpenSci editors. 
+
+```{note}
+In some cases, if the EiC is unable to support a particular package due to 
+conflicts OR if they simple believe another editor is better suited to assess 
+the scope and readiness of a package to be reviewed, they may opt to assign an editor to perform initial checks.
+```
+
+## Editor in Chief checklist
+
+When a new package is submitted for review, the EiC will:
+
+- Tag the issue with `1/editor-checks` tag 
+- Next, they will use the template below in the issue to check whether the package has 
+the bare minimum requirements to initiate a review (or they will assign that task to an editor as stated above). 
+- SURVEY data: To ensure we have information from the authors, please be sure that they filled out our survey. Thank them graciously for doing this. They can skip sections of it if they wish, but we do need their contact information and we do want to track their experience with out review process and organization  
+
+
+### Editor checklist (copy template below to use in the issue)
+
+```markdown
+## Editor checks 
+
+Hi there! Thank you for submitting your package for pyOpenSci
+review. Below are the basic checks that your package needs to pass 
+to begin our review. If some of these are missing, we will ask you 
+to work on them before the review process begins. 
+
+- [ ] **Fit**: The package meets criteria for [fit](https://www.pyopensci.org/contributing-guide/open-source-software-peer-review/aims-and-scope.html#package-categories) and [overlap](https://www.pyopensci.org/contributing-guide/open-source-software-peer-review/aims-and-scope.html#package-overlap).
+- [ ] **Documentation** The package has sufficient documentation available online (README, sphinx docs) to allow us to evaluate package function and scope *without installing the package*. This includes:
+  - [ ] Get started vignettes or tutorials that help a user understand how to use the package and what it can do for them
+  - [ ] API documentation - this includes clearly written doc strings with variables defined using a standard docstring format
+  - [ ] README that clearly articulates the function of the package
+  - [ ] Contributing documentation that details how to install a development environment and how to contribute to the package
+- [ ] **Issue Header Information**: All of the required issue header information is contained within the top of the issue.
```

Comment:
I'm not clear on what "Issue Header Information" is meant to convey. Do you mean like an issue template?

Reply:
Or you mean "the submitter provided all the required info in this submission"?

Reply:
yes i mean they provided all required info... so the YAML header. we can use language that is less technical. i think about parsing a yaml header....

Reply:
changed it to:
**Issue Submission Documentation**: All of the information is filled out in the `YAML` header of the issue (located at the top of the issue template).